src/app/calls.ts

```diff
@@ -0,0 +1,71 @@
+import {AppCallResponse, AppSelectOption} from 'mattermost-redux/types/apps';
+
+class Calls {
+    // createForm returns a form response to create a ticket from a post
+    createForm(message: string): AppCallResponse {
+        const call: AppCallResponse = {
+            type: 'form',
+            form: {
+                title: 'Create Zendesk Ticket',
+                header: 'Create a Zendesk from Mattermost by filling in the required fields and clicking the submit button. Addition text can be added in the `Optional message` form.',
+                footer: 'Message modal form footer',
+                fields: [
+                    {
+                        name: 'subject',
+                        type: 'text',
+                        is_required: true,
+                        hint: 'subject',
```

Comment:
These hints are a bit useless. Maybe get rid of them?

Reply:
Agree.  They are necessary and are a repeat of the displayed field name